projects/project9/package.json

```diff
@@ -5,6 +5,6 @@
     "start": "npm run build && node ./dist/index.js"
   },
   "dependencies": {
-    "@esfx/collections-hashmap": "^1.0.0-pre.17"
+    "buffercursor.ts": "1.0.3"
```

Comment:
Hi @raouldeheer , tell me why did you replace `@esfx/collections-hashmap` with `buffercursor.ts`? I don't know what problem this replacement solves.

Reply:
When running test project 9 in github action it failed [test run](https://github.com/raouldeheer/tsc-alias/actions/runs/3401504758/jobs/5656685500), because npm downloads a newer version of the package that fails the test because of the new exports field. When trying to run the test with the package version pinned to the original version it also fails because it includes other dependencies that result in the same problem. When running the tests locally, it worked because I still had the node_modules folder with the old package (which did not throw any errors). Only after removing node_modules and reinstalling the package the error appeared.

Because the test is demonstrating the use of an alias to a file in node_modules, I assumed that the package used would not matter. So, I replaced it with a package without dependencies with a pinned version, so that the test wouldn't fail because of an error in loading dependencies.

Commit [6a109b2](https://github.com/justkey007/tsc-alias/pull/142/commits/6a109b29bddbac0ca89a26a47348af1151893492) does not have any relevance to the issue therefore could be removed or replaced.